Components/buttons.css

```diff
@@ -0,0 +1,117 @@
+.button {
+  padding: var(--space-xs);
+  outline: none;
+  border: 0px solid transparent;
```

Comment:
If border is 0px, mentioning solid and transparent is not necessary.

Reply:
Thanks for pointing out